server/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ option(onnxruntime_USE_OPENVINO "Build with OpenVINO support" OFF)
 option(onnxruntime_USE_NNAPI_BUILTIN "Build with builtin NNAPI lib for Android NNAPI support" OFF)
 option(onnxruntime_USE_DNNL "Build with DNNL support" OFF)
 option(onnxruntime_USE_NUPHAR "Build with Nuphar" OFF)
+option(onnxruntime_USE_STVM "Build with Stvm" OFF)
```

Comment:
I understand idea, but to rename STVM to TVM is not a short story. Onnx runtime has keys onnxruntime_USE_TVM and USE_TVM which are used by Nuphar. I think it should agree with Microsoft team otherwise it can lead to confusion. Moreover partly this story related to build strategy of ORT with TVM which is not clear for me. In client-oriented scenario it generates many new keys with tag TVM I expected conflicts or confusion with Nuphar. Build system of ORT is complicated enough, I do not want to do it worse for others without clearer understanding of actions.